lib/value/src/value/arbitrary.rs

```diff
@@ -30,13 +30,7 @@ impl Arbitrary for Value {
                 Self::Bytes(Bytes::from(bytes))
             }
             1 => Self::Integer(i64::arbitrary(g)),
-            2 => {
-                let mut f = f64::arbitrary(g) % MAX_F64_SIZE;
-                if f.is_nan() {
-                    f = 0.0;
-                }
-                Self::from(f)
-            }
+            2 => Self::from(f64::arbitrary(g) % MAX_F64_SIZE),
```

Comment:
please revert this part. This will cause tests to fail. The `value` type no longer supports `NaN` values, and will panic if arbitrary picks a `NaN` value.

Reply:
oh, I think there was a bad merge conflict here somewhere. This should be creating a `Value` using the `From<NotNan<f64>>` impl